x/wasm/lbmtypes/proposal.go

```diff
@@ -0,0 +1,77 @@
+package lbmtypes
+
+import (
+	"fmt"
+
+	sdk "github.com/line/lbm-sdk/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
+	govtypes "github.com/line/lbm-sdk/x/gov/types"
+	wasmtypes "github.com/line/lbm-sdk/x/wasm/types"
+)
+
+const (
+	ProposalTypeDeactivateContract wasmtypes.ProposalType = "DeactivateContract"
+	ProposalTypeActivateContract   wasmtypes.ProposalType = "ActivateContract"
+)
+
+var EnableAllProposals = append([]wasmtypes.ProposalType{
+	ProposalTypeDeactivateContract,
+	ProposalTypeActivateContract,
+}, wasmtypes.EnableAllProposals...)
+
+func init() {
+	govtypes.RegisterProposalType(string(ProposalTypeDeactivateContract))
+	govtypes.RegisterProposalType(string(ProposalTypeActivateContract))
+	govtypes.RegisterProposalTypeCodec(&DeactivateContractProposal{}, "wasm/DeactivateContractProposal")
+	govtypes.RegisterProposalTypeCodec(&ActivateContractProposal{}, "wasm/ActivateContractProposal")
+}
+
+func (p DeactivateContractProposal) GetTitle() string { return p.Title }
+
+func (p DeactivateContractProposal) GetDescription() string { return p.Description }
+
+func (p DeactivateContractProposal) ProposalRoute() string { return wasmtypes.RouterKey }
+
+func (p DeactivateContractProposal) ProposalType() string {
+	return string(ProposalTypeDeactivateContract)
+}
+
+func (p DeactivateContractProposal) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(p.Contract); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "contract")
```

Comment:
```suggestion
		return sdkerrors.Wrap(err, "contract")
```

Reply:
I just wanted to notice this request is invalid even though this error is wrong address type.